src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -154,19 +159,29 @@ class ApplicationMenu extends Component<Props> {
     // During onboarding, there is no selected project (because none exists
     // yet). Therefore, we only want to show the 'Project' menu when a project
     // is selected.
-    if (selectedProjectId && devServerTask) {
+    if (selectedProjectId) {
       // The `Project` menu should be inserted right after `Edit`, which will
       // have a different index depending on the platform.
       const editMenuIndex = template.findIndex(menu => menu.id === 'edit');
 
+      // If there's no devServerTask then hide that menu item
+      // this will happen if the project doesn't have any start tasks
+      const clearConsoleMenuItems = devServerTask
```

Comment:
Nit. I think you could write `devServerTask && { ... menuItem code }` here - this will return `undefined` if there is no devServer otherwise it will add the item.

Reply:
I _believe_ it complains if you try to pass `undefined` to a submenu; it doesn't filter out falsy items.

Instead I think I'd do something like this:

```js
let submenu = [];

if (devServerTask) {
  submenu.push(/* clear console item */);
}

submenu.push(/* delete menu item */);
```

And then just use `submenu` in the splice.